content/200-orm/200-prisma-client/100-queries/090-raw-database-access/050-raw-queries.mdx

```diff
@@ -21,12 +21,14 @@ Raw queries are available for all relational databases Prisma ORM supports. In a
 
 ## Raw queries with relational databases
 
-For relational databases, Prisma Client exposes four methods that allow you to send raw queries. You can use:
+For relational databases, Prisma Client exposes four methods that allow you to send raw queries. The methods with "Unsafe" in the name are a lot more flexible but are at **significant risk of making your code vulnerable to SQL injection**. The other two methods are safe to use with a simple template tag, no string building, and no concatenation. However, caution is required for more complex use cases as it is still possible to introduce SQL injection if these methods are used in certain ways. For more details, see the [SQL Injection](#sql-injection) section below.
```

Comment:
Can we move everything starting the second sentence below the list of available queries? I think that would make the page a bit friendlier, while still making sure the potential dangers are highlighted.